public/js/controllers/mempool.js

```diff
@@ -138,8 +184,10 @@
             });
             ws.registerEvtHandler("getmempooltxsResp", (evt) => {
                 this.handleTxsResp(evt)
+                filterVotesLastBlock(this.bestBlockTarget);
+                sortVotesTable();
                 keyNav(evt, false, true)
-            })
+            })    
```

Comment:
No trailing whitespace please. The reason for this is that some contributes have their editors configured to auto delete trailing whitespace, and this muddies up the git history with lines being "changed" with no meaning other than whitespace deletion.